### DISCUSSION TITLE
What is the meaning of when selecting tags always select thr ?

### DISCUSSION THREAD

**Discussion by LipChuanSui** [MaaAssistantArknights/MaaAssistantArknights#7391] [Q&A] [answered]
Hi, What is the meaning of when selecting tags always select thr ?
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/56187131/eaa7dbe2-0fc3-418d-88be-629c1a4eebc4)

**Comment by ABA2396** [comment]
界面没显示全。always select three tags

**Reply by LipChuanSui** [reply]
为什么要选三个？

**Reply by ABA2396** [reply, marked as answer]
比如存在 “快速复活”和“爆发”两个高星词条的时候，maa默认会只选择其中的一组，有人担心会因为被划掉导致失去4星，宁可每次多花一点龙门币来换黄票
